tools/symbol_addrs.txt

```diff
@@ -672,3 +672,18 @@ gspL3DEX2_fifoTextStart = 0x8003FB30;
 gspL3DEX2_fifoDataStart = 0x80045280;
 aspMainTextStart = 0x8003D980;
 aspMainDataStart = 0x80044BA0;
+
+// Unreferenced strings
+D_801C5EDC_98B6FC = 0x801C5EDC; // rom:0x98B6FC type:asciz
+D_801C6054_98B874 = 0x801C6054; // rom:0x98B874 type:asciz
+D_801C6090_98B8B0 = 0x801C6090; // rom:0x98B8B0 type:asciz
+
+D_80208DD8_9D3028 = 0x80208DD8; // rom:0x9D3028 type:asciz
+D_80208E80_9D30D0 = 0x80208E80; // rom:0x9D30D0 type:asciz
+
+D_801EA2C0_A07F80 = 0x801EA2C0; // rom:0xA07F80 type:asciz
+D_801EA4DC_A0819C = 0x801EA4DC; // rom:0xA0819C type:asciz
+D_801EA584_A08244 = 0x801EA584; // rom:0xA08244 type:asciz
+D_801EA68C_A0834C = 0x801EA68C; // rom:0xA0834C type:asciz
```

Comment:
The current version of spimdisasm fails to decode those strings and decides to not migrate them to their corresponding functions. Specifying the type explicitly is a workaround for the migration issue.

This issue will be fixed on a future spimdisasm version but I kept this as a way to ensure the repo builds  OK with older spimdisasm versions